spec/views/candidates/schools/show.html.erb_spec.rb

```diff
@@ -32,6 +32,23 @@
         expect(rendered).to have_css("div.#{css_class} > a", text: 'Start request')
       end
     end
+
+    context 'with availability' do
+      it "has a banner to promote the search page" do
```

Comment:
Think you have your test names the wrong way around

Reply:
Good catch. I pushed a commit to changed it.